src/content/ru/related.mdoc

```diff
@@ -0,0 +1,17 @@
+## Related
+
+### [Расширение Tally]
+
+Расширение Tally для Chrome также доступно для еще более простого подсчета. Просто выделите любой текст, щелкните правой кнопкой мыши и нажмите _**подсчитать**_.
+
+[![Chrome Web Store badge](../../images/web-store-badge.png)][Tally on Chrome Web Store]
+
+### [Mergist - Online PDF Merger][Mergist]
+
+Нужно объединить несколько PDF-файлов? [Mergist] это онлайн-инструмент для объединения нескольких PDF-файлов в один. В Mergist нет рекламы, нет ограничений на размер файлов, и ваши файлы никогда не покидают ваше устройство.
+
+[![Mergist running on an iPhone 15 Pro Max](../../images/mergist-promo.png)][Mergist]
```

Comment:
`Mergist running on an iPhone 15 Pro Max` can be translated as well. This is alt text for screen readers.